src/matlab2tikz.m

```diff
@@ -2724,7 +2728,8 @@ function assertRegularAxes(handle)
         cl = 'specgraph.errorbarseries';
 
     % quiver plots
-        % TODO: maybe check for existence of `udata` and absence of `ldata`
+    elseif hasProperties(h, {'udata','vdata'}, {'ldata'})
```

Comment:
I dont really get this line, as ldata is part of 3D quiverplots, why check for its absence